charts/values.yaml

```diff
@@ -0,0 +1,58 @@
+# -- Number of replicas for the deployment
+replicaCount: 1
+# -- YHS server address
+yhsServerAddr: ":8989"
+
+common:
+  # -- Labels to add to all deployed resources
+  labels: {}
+  # -- Namespace to deploy the resources
+  namespace: "yunikorn"
+
+image:
+  # -- Docker registry
+  registry: "docker.io"
+  # -- Docker image repository
+  repository: "gresearch/yunikorn-history-server"
+  # -- Docker image tag
+  tag: "latest"
+  # -- Docker image pull policy
+  pullPolicy: "IfNotPresent"
+
+service:
+  # -- Service type
+  type: ClusterIP
+  # -- Service port
+  port: 8989
+  # -- Service target port
+  targetPort: 8989
+
+db:
+  # -- YHS database host
+  host: ""
+  # -- YHS database port
+  port: ""
+  # -- YHS database name
+  name: ""
+  # -- YHS database user
+  user: ""
+  # -- YHS database password (insecure, use secrets)
+  password: ""
+  # -- YHS database password secret reference
+  passwordSecretRef: ""
+  # -- Maximum number of connections in the database pool
+  poolMaxConns: 0
```

Comment:
what does 0 mean? unlimited?

Reply:
Actually, I picked up these default values from the `yhs.yaml`